### DISCUSSION TITLE
update

### DISCUSSION THREAD

**Discussion by borderbeater** [3kh0/ext-remover#64] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
it needs an update.

**Comment by 3kh0** [comment]
What do you mean by update?

**Reply by borderbeater** [reply]
the  link for the ext remover is not working i think it needs an update
also thats how i listen to music to calm myself down so i dont beat someone

**Reply by Huntlb** [reply]
go beat thyself

**Comment by CheezeKake77** [comment, marked as answer]
It's probably because your device updated, in which the exploit was patched

**Reply by borderbeater** [reply]
when it happens the thing needs an update
